docs/man_pages/general/resources.md

```diff
@@ -0,0 +1,14 @@
+<% if (isJekyll) { %>---
+title: tns info
+position: 11
```

Comment:
Probably needs a different position - we need to check the rest.

Reply:
I think 9 should be it. Based on the values in https://github.com/NativeScript/nativescript-cli/tree/master/docs/man_pages/project/configuration/resources

And perhaps this file should also be in this folder (not sure how loading of the files is set up, if it would work in that folder)

Reply:
it works well in the resources folder too!